### ISSUE TITLE
feat: Improve filter titles

### ISSUE THREAD

**Issue by oSumAtrIX** [ReVanced/revanced-manager#1998]
### Feature description

If the user is in the patch selector screen,  why does the screen show patches that are unsupported for the selected app?
The filters should say "Not supporting selected version" or "Supporting selected version of the app". Universal should probably be inverted and say "Specifically made for this app":

![image](https://github.com/ReVanced/revanced-manager/assets/13122796/575cb5a0-f711-472c-bcbf-27bd8c20ca5e)

Length of the strings is a problem though, so that needs a solution as well.

### Acknowledgements

- [X] This issue is not a duplicate of an existing feature request.
- [X] I have chosen an appropriate title.
- [X] The feature request is only related to ReVanced Manager

**Comment by X1nto** [comment]
Thoughts about this wording? Supported was renamed to "Supports this version", universal to "Supports all apps". The container has been replaced with a FlowRow to allow wrapping on smaller screens.
![Image](https://github.com/user-attachments/assets/028aa823-c0c7-4361-8a08-90ce158956a0)

**Comment by oSumAtrIX** [comment]
I think vocabulary can be reduced. Patches define a compatibly to apps and versions. ReVanced Manager currently introduces the terminology "supports". Perhaps it's better if the chips use the existing vocabulary to avoid saying the same thing in multiple ways. The chip section title already says compatibility, so the chips can avoid duplication and being too long and just say "This version" and "Any app". Context does the rest

**Comment by X1nto** [comment]
That makes a lot of sense

**Comment by oSumAtrIX** [comment]
I think the filter applies to the apps specific version and not just the app

**Comment by Ushie** [comment]
I was speaking about "Any app"

**Comment by oSumAtrIX** [comment]
Any app filters out universal patches which are compatible with any app

**Comment by Ushie** [comment]
> Any app filters out universal patches which are compatible with any app. This app would imply a filter for just this current apl which doesn't make sense

Like "This version", it can be toggled on as the default state

**Comment by oSumAtrIX** [comment]
Yes but an This app chip would imply toggling patches for this apl, we want a toggle for universal patches, not patches for this app
